lasso/utils/rich_progress_bars.py

```diff
@@ -67,6 +70,7 @@ def render(self, task: Any) -> str:
 
 
 class SubsamplingWaitTime(ProgressColumn):
+    """ *add description? """

```

Comment:
Simply add `TODO` next time as it is a universal flag easily found in any codebase.

Reply:
I am quite new working with public repos. with adding TODO, do you mean writing the keyword inside the quotes or is it some kind of tag?

Reply:
Yeah writing simply the keyword. Otherwise we can also add a linting excuse if it does not look important.